lucien-ios/Lending/LendABookViewController.swift

```diff
@@ -0,0 +1,153 @@
+//
+//  LendABookViewController.swift
+//  lucien-ios
+//
+//  Created by Ahmed, Guled on 10/19/17.
+//  Copyright © 2017 Intrepid Pursuits. All rights reserved.
+//
+
+import UIKit
+import RxSwift
+import RxCocoa
+
+class LendABookViewController: UIViewController, UITableViewDelegate, AlertDisplaying {
+
+    // MARK: - Private IBOutlets
+
+    @IBOutlet private weak var comicPhoto: UIImageView!
+    @IBOutlet private weak var seriesTitleLabel: UILabel!
+    @IBOutlet private weak var storyTitleLabel: UILabel!
+    @IBOutlet private weak var volumeLabel: UILabel!
+    @IBOutlet private weak var tableView: UITableView!
+    @IBOutlet private weak var comicInfoView: UIView!
+
+    // MARK: - Private Constants
+
+    private let disposeBag = DisposeBag()
+    private let viewModel = LendingBookViewModel()
+
+    // MARK: - Private Variables
+
+    private var nextButton = UIBarButtonItem()
+    private let lendingCell = UINib(nibName: "LendingTableViewCell", bundle: nil)
+    private let headerView = UIView()
+
+    // MARK: - Variables
+
+    var selectedIndex: Int?
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        configureNavigationController()
+        configureTableView()
+
+        viewModel.getAllUsers { [weak self] (users, error) in
+            if error == nil {
+                guard let users = users else { return }
+                self?.viewModel.users.value = users
+                DispatchQueue.main.async {
+                    self?.tableView.reloadData()
+                }
+            } else {
+                self?.showAlert(title: "Error", message: "Our service is currently encountering an issue. Please ensure that you are connected to the internet and try again.")
```

Comment:
You could make use of the error you get here to maybe give the user more info? Maybe it's outside the scope of this work though.

Reply:
There is no specific error for this particular endpoint. If I still have some time left over, after completing the lending feature, I'll make sure to use the errors given by the server.